flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/JobManagerRunnerTest.java

```diff
@@ -64,7 +64,7 @@
 import static org.junit.Assert.fail;
 
 /**
- * Tests for the {@link JobManagerRunner}.
+ * Tests for the {@link JobManagerRunnerImpl}.
  */
 public class JobManagerRunnerTest extends TestLogger {
```

Comment:
I think it can be renamed it to `JobManagerRunnerImplTest`.

Reply:
True, will change it.